### PR TITLE
Compile scss in order to ship the CSS file and fix the import js path

### LINKED ISSUE #22: We should have a compiled version of the scss

### LINKED ISSUE #24: Fix Histogram import path

### PR DESCRIPTION
**Pull request by victorfern91**
Closes #24  Closes #22

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Brushable Histogram
 
-## 1.1.6 (2018/12/26)
+## 1.2.0 (TBD)
+- Export a css file
```
Comment (lfac-pt) [inline CHANGELOG.md]: We should put the verbs all similar, like "Exports" "Ignores" "Updates"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
 # Brushable Histogram
 
-## 1.1.6 (2018/12/26)
+## 1.2.0 (TBD)
```
Comment (lfac-pt) [inline CHANGELOG.md]: Do we really need to put this as a minor? We actually didn't add any new functionality (which is what defined a minor)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+- Export a css file
+- Ignoring directories that are irrelevant for this component releases.
+  Like unit tests, documentation and storybook files.
+- Update .babelrc file in order to ship the source code without unit
+  tests.
+- Fix Javascript import path.
 - Adds missing check that cause the histogram to blow-up in some cases
 
-## 1.1.5 (2018/12/26) BUMPED
+## 1.1.5 & 1.1.6 (2018/12/26) BUMPED
```
Comment (lfac-pt) [inline CHANGELOG.md]: 1.1.6 was actually removed, no? Why do we have it here?

File: CHANGELOG.md (hunk 2 above)
Comment (victorfern91) [reply CHANGELOG.md]: You're right

File: CHANGELOG.md (hunk 3 above)
Comment (victorfern91) [reply CHANGELOG.md]: I've reverted that
image

Comment (lfac-pt) [reply CHANGELOG.md]: Yes, we removed it, so it shouldn't be here in the changelog, right?

Comment (lfac-pt) [reply CHANGELOG.md]: I think it makes more sense to delete the tag and have it consistent between the repository and npm

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+## 1.1.6 (TBD)
+- Exports a css file
+- Ignoring directories that are irrelevant for this component releases.
+  Like unit tests, documentation and storybook files.
+- Updates .babelrc file in order to ship the source code without unit
+  tests.
+- Fixes Javascript import path.
 - Adds missing check that cause the histogram to blow-up in some cases
 
 ## 1.1.5 (2018/12/26) BUMPED
```
Comment (lfac-pt) [inline CHANGELOG.md]: BUMPED?

Comment (lfac-pt) [reply CHANGELOG.md]: The "bump" is still here :P

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@feedzai/brushable-histogram",
-  "version": "1.1.6",
+  "version": "1.1.5",
```
Comment (lfac-pt) [inline package.json]: +1

**Pull request merged**